From 6c49d5ef57c11fe82587dd08d5ca213df968d111 Mon Sep 17 00:00:00 2001
From: "Pan, Kris" <kris.pan@intel.com>
Date: Tue, 25 Aug 2020 15:08:04 +0800
Subject: [PATCH 1204/1214] soundwire: fix sdw_release_stream export warning

WARNING: drivers/soundwire/soundwire-bus: 'sdw_release_stream' exported twice.
Previous export was in drivers/sdw/sdw_bus.ko

Signed-off-by: Pan, Kris <kris.pan@intel.com>
---
 drivers/soundwire/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soundwire/Kconfig b/drivers/soundwire/Kconfig
index 73d6ffe..4daea6c 100644
--- a/drivers/soundwire/Kconfig
+++ b/drivers/soundwire/Kconfig
@@ -4,7 +4,7 @@
 
 menuconfig SOUNDWIRE
 	tristate "SoundWire support"
-	depends on ACPI && on !SDW
+	depends on SDW=n && ACPI
 	help
 	  SoundWire is a 2-Pin interface with data and clock line ratified
 	  by the MIPI Alliance. SoundWire is used for transporting data
-- 
2.7.4

